app/models/category.rb

```diff
@@ -47,5 +50,8 @@ def read_translated_attribute(name)
     globalize.stash.contains?(Globalize.locale, name) ? globalize.stash.read(Globalize.locale, name) : translation_for(Globalize.locale).send(name)
   end
 
+  def is_not_common_replies?
```

Comment:
I think for future extensibilty it might be better to use a more generic name- ie. `non_deleteable?` or `protected?`  That way in the future if there are other categories to be protected, they can be added to this method.